snapshot-labs/snapshot
# feat: add og subgraph queries

### Issues
we recently deployed an OG subgraph. We want to add queries to it to further optimize calls when requesting historical data.

Fixes #

### Changes 
*_(Briefly describe the changes made in this PR)_*

1. this removes all the event based queries and replaces them with calls to the OG subgraph
2. this also fixes an issue with incorrect data type being returned from subgraph query, number changed to bignumber
3. this also updates to production subgraph points, and updates any missing contract addresses in constants

### How to test
*_(Explain how the changes can be tested, including any required setup steps)_*

1. go to the proposal: #/barnbridge.eth/proposal/0x74cf6f3372aaeebfc2e99ff54604a5e63ca3d44cb84bdbf202a85581dfac9f5d
see that there is no warning in console Error querying module details from the graph. this means it used the subgraph function successfully. it shows that it has been executed.

### To-Do
*_(List any outstanding tasks be addressed before or after this PR is merged)_*

### Self-review checklist

### Additional notes or considerations
*_(Include any other relevant information or context that may be helpful for the reviewer)_*
